piecewise_web/index.html

```diff
@@ -401,17 +406,28 @@ <h3 class="league-gothic">Thank you!</h3>
 map.scrollWheelZoom.disable();
 var control = L.control.zoom({position: 'topright'});
 map.addControl(control);
-var osmLayer = L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', {
-	attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a>' +
-		'contributors'
-});
 
+// Use Open Street Maps as a base tile layer
+// var osmLayer = L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', {
+//	attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a>' +
+//		'contributors'
+// });
+
+// Use Mapbox as a base tile layer
+// var mapboxLayer = L.tileLayer(
+//		'https://{s}.tiles.mapbox.com/v3/newamerica.lcl1jan5/{z}/{x}/{y}.png', {
+//	attribution: '&copy; <a href="http://mapbox.com/">Mapbox</a>'
+// });
+ 
+// Use Mapbox as a base tile layer
 var mapboxLayer = L.tileLayer(
 		'https://{s}.tiles.mapbox.com/v3/newamerica.lcl1jan5/{z}/{x}/{y}.png', {
 	attribution: '&copy; <a href="http://mapbox.com/">Mapbox</a>'
 });
 
-// Set the default base tile layer
+// Set the default base tile layer. 
+// If using Open Street Maps: map.addLayer(osmLayer);
```

Comment:
I confirmed that the instructions in config.md and code comments in index.html on lines 410-420 and lines 428-430 provide instructions on commenting/uncommenting the appropriate layer code to use the base layer of their choice.